core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -307,6 +314,65 @@ default Iterator<TableColumnsMetadata> streamTableColumns(ConnectorSession sessi
                 .iterator();
     }
 
+    /**
+     * Gets comments for all relations (tables, views, materialized views) that match the specified prefix
+     * (e.g. for all relations that would be returned by {@link #listTables(ConnectorSession, Optional)} when
+     * {@code prefix} does not represent relation name).
```

Comment:
This method doesn't take "prefix" as the method argument. Should we reword the sentence?

Reply:
thanks, that's a left over. good catch!